pyshelf/link_manager.py

```diff
@@ -31,9 +31,6 @@ def assign_listing(self, path_list):
                 rel_type = "collection"
                 title = "a collection of artifacts"
 
-            if resource_path == self.request.path:
```

Comment:
This doesn't seem right.  Do we never assign self?  What about if we do a head request on an artifact?  Is this handled elsewhere?